### ISSUE TITLE
Property strings of extensions aren't translated 

### ISSUE THREAD

**Issue by SusanRatiLane** [mit-cml/appinventor-sources#3258]
**Describe the bug**

Property strings of extensions are not translated in the help widget on the Properties sidebar of the designer. The same strings are translated correctly in the blocks for those same properties.

**Affects**

- [X] Designer
- [ ] Blocks editor
- [ ] Projects Explorer
- [ ] Companion
- [ ] Compiled apps
- [ ] Buildserver
- [ ] Debugging
- [ ] Other... (please describe)

**Comment by YashSachdeva** [comment]
I think Designer's Properties sidebar and the Blocks Editor handle translations differently. While the Blocks Editor fetches property strings using a proper translation mechanism, the Designer seems to miss linking the property strings to the correct localization resources.

**Comment by SusanRatiLane** [comment]
> I think Designer's Properties sidebar and the Blocks Editor handle translations differently. While the Blocks Editor fetches property strings using a proper translation mechanism, the Designer seems to miss linking the property strings to the correct localization resources.

Yes, this is a known issue on my plate to fix. It might not have a github issue, though.

**Comment by josmas** [comment]
Screenshot shared in community in this thread: https://community.appinventor.mit.edu/t/helpurl-not-displayed/172587
